src/main/java/sirius/kernel/Setup.java

```diff
@@ -58,7 +58,26 @@
      * Determines the mode in which the framework should run. This mainly effects logging and the configuration.
      */
     public enum Mode {
-        DEV, TEST, PROD
+
+        /**
+         * Defines the IDE development {@link Mode}
```

Comment:
why link to the enum itself. rather use one sentence per value what the environment typically is - esp. staging vs. test etc.

Reply:
Thats right, changed it.